swift_scality_backend/http_utils.py

```diff
@@ -92,9 +93,14 @@ class HTTPResponse(httplib.HTTPResponse):
         '''
         def __init__(self, sock, debuglevel=0, strict=0, method=None,
                      buffering=False):
-            httplib.HTTPResponse.__init__(self, sock, debuglevel=debuglevel,
-                                          strict=strict, method=method,
-                                          buffering=buffering)
+            version = sys.version_info[:2]
+            if version == (2, 6):
```

Comment:
It might make sense to turn this into a single `__init__` call, passing in a `**dict`:

```python
init_args = {
    'sock': sock,
    'debuglevel': debuglevel,
    'strict': strict,
    'method': method,
}

if sys.version_info >= (2, 7):
    init_args.update({
        'buffering': buffering,
    })

httplib.HTTPResponse.__init__(self, **init_args)
```